lib/plugins/wallet/pazuz-wallet/pazuz-wallet.js

```diff
@@ -0,0 +1,108 @@
+const https = require('https')
+const BN = require('../../../bn')
+const E = require('../../../error')
+
+const axios = require('axios').create({
+  // TODO: get rejectUnauthorized true to work
+  baseURL: 'https://localhost:5555/api/',
+  httpsAgent: new https.Agent({
+    rejectUnauthorized: false
+  })
+})
+
+const SUPPORTED_COINS = ['BTC', 'ZEC', 'LTC', 'BCH', 'DASH', 'ETH']
+
+function checkCryptoCode (cryptoCode) {
+  if (!SUPPORTED_COINS.includes(cryptoCode)) {
+    return Promise.reject(new Error('Unsupported crypto: ' + cryptoCode))
+  }
+
+  return Promise.resolve()
```

Comment:
Could be solved with a ternary. Plus, something like lodash should be used, instead of the vanilla JS library. Rafael told me that a while ago, so you should check if that still applies